docs/csharp/programming-guide/interop/how-to-access-office-onterop-objects.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "How to: Access Office Interop Objects by Using Visual C# Features - C# Programming Guide"
+title: "How to access Office interop objects - C# Programming Guide"
```

Comment:
Removed `by Using Visual C# Features` because the title already have the `"C# Programming Guide"` part.